rice-middleware/kew/api/src/main/java/org/kuali/rice/kew/api/util/UserTextFilterForXml.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2006-2015 The Kuali Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.opensource.org/licenses/ecl2.php
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kuali.rice.kew.api.util;
+
+import com.sun.org.apache.xml.internal.utils.XMLChar;
```

Comment:
Using com.sun imports is bad since it is not guaranteed to work on other jvm's. Is there another option for this XMLChar class?